internal/dbtools/hooks.go

```diff
@@ -392,20 +392,31 @@ func AuditGroupMemberPromoted(ctx context.Context, exec boil.ContextExecutor, pI
 }
 
 // AuditGroupMembershipApproved inserts an event representing group membership approval into the events table
-func AuditGroupMembershipApproved(ctx context.Context, exec boil.ContextExecutor, pID string, actor *models.User, m *models.GroupMembership) ([]*models.AuditEvent, error) {
+func AuditGroupMembershipApproved(ctx context.Context, exec boil.ContextExecutor, pID string, actor *models.User, m *models.GroupMembership, kind string) ([]*models.AuditEvent, error) {
 	// TODO non-user API actors don't exist in the governor database,
 	// we need to figure out how to handle that relationship in the audit table
 	var actorID null.String
 	if actor != nil {
 		actorID = null.StringFrom(actor.ID)
 	}
 
+	var action string
+
+	switch kind {
+	case "new_member":
+		action = "group.member.request.approved"
+	case "admin_elevation":
+		action = "admin.elevation.request.approved"
```

Comment:
you've probably used this everywhere but why not promotion/demotion?

Reply:
Because... I thought of this word first ha. That would be more consistent and not hard to change - I'll update it.